infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/processors/ProcessorDefinition.java

```diff
@@ -0,0 +1,104 @@
+package com.redhat.service.smartevents.infra.v2.api.models.processors;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+//import com.redhat.service.smartevents.infra.v1.api.models.filters.BaseFilter;
+//import com.redhat.service.smartevents.infra.v1.api.models.gateways.Action;
+//import com.redhat.service.smartevents.infra.v1.api.models.gateways.Source;
+
+public class ProcessorDefinition {
+
+    //    @JsonProperty("filters")
+    //    private Set<BaseFilter> filters;
+
+    @JsonProperty("transformationTemplate")
+    private String transformationTemplate;
```

Comment:
Are transformations in v2?

I thought this would be a list of Flows? (or at least a Json `ObjectNode`)